junction/proposals/models.py

```diff
@@ -27,6 +29,8 @@ class ProposalSection(AuditModel):
     description = models.TextField(default="")
     active = models.BooleanField(default=True, verbose_name="Is Active?")
     conferences = models.ManyToManyField(to=Conference, related_name='proposal_sections')
+    start_date = models.DateField(default=datetime.now, verbose_name="Start Date")
+    end_date = models.DateField(default=datetime.now, verbose_name="End Date")
```

Comment:
We should these two fields in admin page as well right https://github.com/pythonindia/junction/blob/master/junction/proposals/admin.py#L25 ?